docs/quick.rst

```diff
@@ -34,10 +34,13 @@ when using h5py is:
 
     **Groups work like dictionaries, and datasets work like NumPy arrays**
 
-Suppose someone has sent you a HDF5 file, :code:`mytestfile.hdf5`. (To create this file, read `Appendix: Creating a file`_.) The very first thing you'll need to do is to open the file for reading::
+Suppose someone has sent you a HDF5 file, :code:`mytestfile.hdf5`. (To create
+this file, follow the commands in `Appendix: Creating a file`_.) The very first
+thing you'll need to do is to open the file for reading and writing (omit the
+:code:`'a'` or replace it with the option :code:`'r'` to read only)::
 
     >>> import h5py
-    >>> f = h5py.File('mytestfile.hdf5', 'r')
+    >>> f = h5py.File('mytestfile.hdf5', 'a')
```

Comment:
I'm not sure it makes sense to open read-write here, as this first section is all about reading.

If we're assuming someone follows this page all the way through, they'll reopen `f` read-write in the 'creating a file' section before any write operations. If we're assuming someone jumps to the section they're interested in, then the current structure which points out the switch to read-write mode seems preferable. The current structure of the page isn't really clear what pattern it's written for, though - maybe we need a bigger overhaul to rearrange it into a reading part and a writing part. :shrug: